### DISCUSSION TITLE
Helpers module

### DISCUSSION THREAD

**Discussion by sheilaaliens** [RescueSocialTech/Amber-Heard_Disinformation_Operations_Bots#8] [Q&A]
When I try to run any of this in Google colabs get an error saying there’s no module named helpers. Please help

**Comment by christinataft** [comment]
Hi Sheilaaliens,

The helpers modules is part of @pe-mn's work on Threat Analysis. https://github.com/RescueSocialTech/Amber-Heard_Disinformation_Operations_Bots/search?q=helpers
I will ask about adding the helpers to this. There are also some helpers referenced from @adelabuhashim on Links Analysis.
-- I see a helpers.py file here under Wrangling & Analysis, Facebook Analysis - https://github.com/RescueSocialTech/Amber-Heard_Disinformation_Operations_Bots/tree/main/_Facebook%20Analysis/Wrangling%20%26%20Analysis

Thank you for your question and testing our project which makes it better!

**Reply by adelabuhashim** [reply]
Each part of analysis has its own "helpers.py" file.
The repository will adapted  to has all those "helpers" files in the proper place to run you notebooks without any problems!

**Reply by christinataft** [reply]
@sheilaaliens - Adel will be working on refactoring the code for coders like yourself to be able to work on it without needing to edit. It will be in a folder of "Useable Code - Refactored" to work for all parts of our repository.

I organized it the way that I did so that also non-coders can understand it. It's incredibly hard to translate what we did to all parties. Thanks again for following our project and we hope it can help others too...

**Comment by royomachinery** [comment]
Unprecedented global Social Network Analysis on Amber Heard reveals a Category 6 disinformation operation spanning 2018-2022. Millions of texts, accounts, and 8 platforms studied by a team of 20+ researchers for over a year. Data available for public scrutiny, aiming to set precedents and foundations for combating cyber abuse, disinformation, and improving lives. Ongoing project invites collaboration and contributions from researchers, ethical public, and students worldwide.
